omnibus/Gemfile.lock

```diff
@@ -281,6 +284,11 @@ GEM
       ffi-win32-extensions (~> 1.0.3)
       win32-process (~> 0.9)
       wmi-lite (~> 1.0)
+    mixlib-shellout (3.2.7-x64-mingw-ucrt)
```

Comment:
Are you sure that chef-17 should have ucrt?

Reply:
it was definitely the one backed out but I would think it should be `universal-mingw32` instead

Reply:
`mixlib-shellout (3.2.7-universal-mingw32)` is already there intact right before this one in the Gemfile.lock. Perhaps the build error occurs as the Ruby toolkit running for chef-17 now is UCRT.

Reply:
the build error in log indicates that omnibus-toolchain, which is now UCRT is requiring the relevant gem
```
C:/opscode/omnibus-toolchain/embedded/lib/ruby/gems/3.1.0/gems/mixlib-shellout-3.2.7/lib/mixlib/shellout/windows.rb:21:in `require': cannot load such file -- win32/process (LoadError)
```